### PR TITLE
remove verify apo

### PR DESCRIPTION
**Pull request by peetucket**
## Why was this change made?

to deal with sul-dlss/argo#1801 by eliminating the verify-apo robot

To deploy this change, we also need to:

- [ ] remove verify-apo robot from robot config in shared_config https://github.com/sul-dlss/shared_configs/blob/preservation_robots_prod/config/environments/robots_prod.yml
- see https://github.com/sul-dlss/shared_configs/pull/1335
- [ ] remove verify-apo robot from robot config in shared_config https://github.com/sul-dlss/shared_configs/blob/preservation_robots_stage/config/environments/robots_stage.yml
- see https://github.com/sul-dlss/shared_configs/pull/1334
- [ ] Remove workflow step from template: https://github.com/sul-dlss/workflow-server-rails/blob/master/config/workflows/preservationIngestWF.xml#L14 
- see sul-dlss/workflow-server-rails#373
- [ ] In the workflow db, set any verify-apo steps that are not status=completed to status=skipped:
```
WorkflowStep.where(workflow: 'preservationIngestWF', process: 'verify-apo').incomplete.size # show how many rows
WorkflowStep.where(workflow: 'preservationIngestWF', process: 'verify-apo').incomplete.each {|step| step.update(status: 'skipped')} # update all to skipped; note error steps will be set to skipped as well
```

## Was the usage documentation (e.g. README, DevOpsDocs, wiki, queue specific README) updated?

could not find any documentation that references this robot specifically

## Does this change affect how this application integrates with other services?

If so, please confirm:
- change was tested on stage    and/or
- test added to sul-dlss/infrastructure-integration-test

### CODE REVIEW

**Comment by peetucket** [comment]
> Do you need to remove the step from the workflow-server, or has that already been done?

That's not done yet.  Its just editing this file, right?  https://github.com/sul-dlss/workflow-server-rails/blob/master/config/workflows/preservationIngestWF.xml

I can put in a  PR for that.  

I presume it makes sense to remove that first.

**Comment by peetucket** [comment]
sul-dlss/workflow-server-rails#373

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -126,7 +117,6 @@ RSpec/MessageSpies:
     - 'spec/preservation_ingest/complete_ingest_spec.rb'
     - 'spec/preservation_ingest/transfer_object_spec.rb'
     - 'spec/preservation_ingest/update_moab_spec.rb'
-    - 'spec/preservation_ingest/verify_apo_spec.rb'
```
Comment (ndushay) [inline .rubocop_todo.yml:129]: haha - did you hand edit this?  the counts didn't go down.  And it doesn't matter.

Comment (peetucket) [reply .rubocop_todo.yml:129]: yes, guilty

a full regen was a lot of changes

**Pull request merged**